elastic/kibana
# [Inventory] Remove `Actions` tooltip

## Summary

Closes https://github.com/elastic/kibana/issues/199500

This PR removes the Actions tooltip in the Entities Table.

## Screenshots
| Before                                                                                    | After                                                                                     |
|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
|![image](https://github.com/user-attachments/assets/c661245a-7d07-4203-b832-a59acc20259c)|<img width="1441" alt="image" src="https://github.com/user-attachments/assets/29884aa1-3d82-4e0e-ae88-222fc8edf438">|

## How to test
1. Go to Inventory
2. Check the columns of the Entities table, the tooltip + help button should now be gone.
